src/Components/Publishing/Display/__tests__/DisplayPanel.test.tsx

```diff
@@ -106,6 +107,10 @@ describe("units", () => {
       ])
     })
 
+    it("checks for enabled ads", () => {
+      expect(isHTLAdEnabled()).toBe(false)
+    })
```

Comment:
Same here -- this can be moved into it's own test file.

What should be tested inside of `DisplayPanel` is that an ad is injected, and that insertion works correctly.